GraspObject object
geometry_msgs/PoseStamped pose
---
bool is_valid